packages/cascara/src/ui/Form/Form.js

```diff
@@ -162,6 +167,7 @@ const Form = ({
   onAction,
   isEditable: incomingIsEditable,
   isInitialEditing = false,
+  isLoading = false,
```

Comment:
For isLoading we can try using the getStateFromData function to infer isLoading and isEmpty. That function however probably needs to get updated to work with objects as I think it is only working with arrays.

const { isLoading } = getStateFromData(data)

Reply:
@brianespinosa IDK if we moved the `getStatusFromDataLength` util from the monorepo, just in case here it is so @rociobaena can use it in Cascara:

```javascript
const getStatusFromDataLength = (totalRecordCount) => {
  const isLoading = !totalRecordCount && totalRecordCount !== 0;
  const isEmpty = !isLoading && totalRecordCount === 0;

  return {
    isEmpty,
    isLoading,
  };
};

export default getStatusFromDataLength;
```

Reply:
@rociobaena I think the function @lordpixel is mentioning above was actually moved already because Pagination is officially in Cascara now. ;)

Reply:
Using getStatusFromDataLength instead.